pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -44,6 +47,7 @@ class ConnectionRepository(object):
     _pywbemcli_servers = {}
     _loaded = False
     _connections_file = None
+    yaml_connections_group = 'connection-definitions'
```

Comment:
I suggest to always use underscores for property names in YAML (or JSON), and not hyphens. That makes creating dicts in Python with such properties simpler, for example because dicts can be initialized with the property names as named arguments, e.g. `dict(connection_definitions=dict())`.

Reply:
No issue here.